subsys/disk/disk_access_spi_sdhc.c

```diff
@@ -16,20 +18,28 @@ LOG_MODULE_REGISTER(sdhc_spi, CONFIG_DISK_LOG_LEVEL);
 #include "disk_access_sdhc.h"
 
 /* Clock speed used during initialisation */
-#define SDHC_SPI_INITIAL_SPEED 400000
+#define SDHC_SPI_INITIAL_SPEED KHZ(400)
 /* Clock speed used after initialisation */
-#define SDHC_SPI_SPEED 4000000
+#define SDHC_SPI_DEFAULT_SPEED MHZ(25)
```

Comment:
25 should come from somewhere. Not sure if it should come from DT_* or Kconfig. Last resort is having it as a function parameter.